spec/models/task_spec.rb

```diff
@@ -1,5 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
+    it 'is valid with all attributes' do
+      user = FactoryBot.create(:user)
+      task = FactoryBot.build(:task, user: user)
+      expect(task).to be_valid
+      expect(task.errors).to be_empty
+    end
+
+    it 'is invalid without title' do
+      user = FactoryBot.create(:user)
+      task = FactoryBot.build(:task, title: "",  user: user)
+      expect(task).to be_invalid
+      expect(task.errors[:title]).to eq ["can't be blank"]
+    end
+
+    it 'is invalid without status' do
+      user = FactoryBot.create(:user)
+      task = FactoryBot.build(:task, status: nil, user: user)
```

Comment:
どこまで細かくするかによりますが、役割が分かる名前にしておきましょうか。
```suggestion
      task_without_status = FactoryBot.build(:task, status: nil, user: user)
```

Reply:
5744b42